applications/feedback_sentiment_analyzer/cdk/lib/stack.ts

```diff
@@ -0,0 +1,136 @@
+import { Stack, CfnOutput } from "aws-cdk-lib";
+import { Construct } from "constructs";
+
+import { APP_LANG, APP_EMAIL, PREFIX } from "./env";
+import { AppLambdas } from "./constructs/app-lambdas";
+import { getFunctions as getFunctionConfigs } from "./functions";
+import { AppStateMachine } from "./constructs/app-state-machine";
+import { AppCloudFrontWebsite } from "./constructs/app-cloud-front-website";
+import { AppEnvLambda } from "./constructs/app-env-lambda";
+import { Cors, RestApi } from "aws-cdk-lib/aws-apigateway";
+import {
+  CachePolicy,
+  Distribution,
+  OriginRequestPolicy,
+  ResponseHeadersPolicy,
+  ViewerProtocolPolicy,
+} from "aws-cdk-lib/aws-cloudfront";
+import { RestApiOrigin, S3Origin } from "aws-cdk-lib/aws-cloudfront-origins";
+import { AppAuth } from "./constructs/app-auth";
+import { AppRoutes } from "./constructs/app-routes";
+import { Empty, EnvModel } from "./constructs/app-api-models";
+
+export class AppStack extends Stack {
+  constructor(scope: Construct) {
+    const prefix = `fsa-${PREFIX}`;
+    super(scope, prefix);
+
+    // Create AWS Lambda functions.
+    const fnConfigs = getFunctionConfigs(APP_LANG);
+    const appLambdas = new AppLambdas(this, "fn", fnConfigs);
+
+    // Create state machine.
+    const appStateMachine = new AppStateMachine(
+      this,
+      prefix,
+      appLambdas.functions
+    );
+
+    // Create API
+    // const logGroup = new LogGroup(this, `api-log-group`);
+    const api = new RestApi(this, `${prefix}-api`, {
+      defaultCorsPreflightOptions: {
+        allowOrigins: Cors.ALL_ORIGINS,
+        allowCredentials: true,
+      },
+      deployOptions: {
+        // accessLogDestination: new LogGroupLogDestination(logGroup),
+        // accessLogFormat: AccessLogFormat.jsonWithStandardFields(),
+      },
+    });
+
+    // Create API routes.
+    const routes = new AppRoutes(this, `${prefix}-routes`, {
+      api,
+    });
+
+    // Create static S3 website behind a CloudFront distribution.
+    const website = new AppCloudFrontWebsite(this, "client", {
+      assetPath: "../client",
+    });
+
+    // Create CloudFront distribution
+    const s3Origin = new S3Origin(website.bucket);
+    const s3OriginConfig = {
+      origin: s3Origin,
+      viewerProtocolPolicy: ViewerProtocolPolicy.ALLOW_ALL,
+      cachePolicy: CachePolicy.CACHING_OPTIMIZED,
+      originRequestPolicy: OriginRequestPolicy.ALL_VIEWER_EXCEPT_HOST_HEADER,
+    };
+    const distribution = new Distribution(this, "website-distribution", {
+      defaultRootObject: "index.html",
+      defaultBehavior: s3OriginConfig,
+      additionalBehaviors: {
+        "/*.js": {
```

Comment:
css, as well

Reply:
<img width="1181" alt="Screenshot 2023-06-20 at 1 24 12 PM" src="https://github.com/awsdocs/aws-doc-sdk-examples/assets/2723491/680cb71b-c24c-431d-a397-84639854a900">
CSS seems to work automatically